README.md

```diff
@@ -1,75 +1,49 @@
-![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# Project Name 
+To do list.
 
-> Description the project.
+# Description of project.
+This To-do list App is built using webpack and served by a webpack dev server
```

Comment:
- [ ] [OPTIONAL] Can you please also add what the Todo list app can do the description e,g add, remove, edit tasks. This will give the user information on what the Todolist app can do